cli/Cargo.toml

```diff
@@ -28,3 +28,15 @@ assert_cmd = "2"
 default = ["native"]
 # Non-wasi interface. These features cannot be compiled to WASI.
 native = ["edit"]
+
+[package.metadata.deb]
+extended-description = "Atomic CLI - A command-line interface for headless CMS and real-time databases."
+maintainer = "tomoyan <https://www.tomoyan.net/>"
+copyright = "2025, tomoyan"
```

Comment:
No this is not how copyright works :P I wrote pretty much the entire repo, you can't claim ownership with one PR that adds some metadata.